docs/smart_rollups.md

```diff
@@ -29,7 +29,6 @@ await op.confirmation();
 - `pvmKind` at this time the only PVM supported is `wasm_2_0_0`
 - `kernel` is passed as a hexadecimal string examples can be found at [this tezos docs endpoint](https://tezos.gitlab.io/mumbai/smart_rollups.html?highlight=smart%20rollup%20originate#id1)
```

Comment:
link is 404 might want to update it to `https://tezos.gitlab.io/active/smart_rollups.html`

Reply:
good catch, fixing